static/js/zcommand.js

```diff
@@ -94,6 +94,12 @@ exports.process = function (message_content) {
         return true;
     }
 
+    if (content === '/settings') {
+        settings.setup_page();
+        settings.launch_page("settings");
```

Comment:
This opens the settings overlay, but doesn't change the url to `#settings/your-account`.

Reply:
Where as `window.location.hash = 'settings/your-account'` changes the url and opens the overlay.